manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -97,6 +97,11 @@ sidecarInjectorWebhook:
   # Only one environment should have this enabled.
   enableNamespacesByDefault: false
 
+  # Mutations that occur after the sidecar injector are not handled by default, as the Istio sidecar injector is only run
+  # once. For example, an OPA sidecar injected after the Istio sidecar will not have it's liveness/readiness probes rewritten.
+  # Setting this to `IfNeeded` will result in the sidecar injector being run again if additonal mutations occur.
+  reinvocationPolicy: Never
```

Comment:
Do we want to consider changing the default? I suppose that can be a followup  - I _think_ its probably desireable but concerned about the unknown-unknowns

Reply:
I think for now it's best to leave it as `Never` since that keeps the behavior the same as before.